src/storage.rs

```diff
@@ -70,6 +70,7 @@ impl GetEntriesContext {
     pub fn can_async(&self) -> bool {
         match self.0 {
             GetEntriesFor::SendAppend { .. } => true,
+            GetEntriesFor::SendForward { .. } => true,
```

Comment:
I think it can be false to avoid the async fetch's potential latency in follower replication.